airflow/providers/amazon/aws/sensors/rds.py

```diff
@@ -149,7 +161,54 @@ def poke(self, context: 'Context'):
         return self._check_item(item_type='export_task', item_name=self.export_task_identifier)
 
 
+class RdsDbSensor(RdsBaseSensor):
```

Comment:
2022/09/05 update (https://github.com/apache/airflow/pull/26003/commits/0d537b1bdf1fa3dbd8a94f1b635a5814e84e11e9)
Renamed `RdsInstanceSensor` to `RdsDbSensor` because it considers "cluster" databases as well